src/main/resources/edu/cornell/kfs/vnd/cu-ojb-vnd.xml

```diff
@@ -253,7 +253,99 @@
     </collection-descriptor>  -->
 </class-descriptor>
 
-<class-descriptor class="org.kuali.kfs.vnd.businessobject.VendorAddress" table="PUR_VNDR_ADDR_T">
+    <class-descriptor class="org.kuali.kfs.vnd.businessobject.VendorTaxDetail" table="PUR_VNDR_DTL_T">

```

Comment:
Is VendorTaxDetail meant to share the same table as VendorDetail? Is that how it's currently set up in KFS 6?